llmware/library.py

```diff
@@ -432,7 +586,55 @@ def add_files (self, input_folder_path=None, encoding="utf-8",chunk_size=400,
                    verbose_level=2, copy_files_to_library=True):
 
         """ Main method to integrate documents into a Library - pass a local filepath folder and all files will be
```

Comment:
Delete space before you start the doc.
```suggestion
        """Main method to integrate documents into a Library - pass a local filepath folder and all files will be
```